frontend/src/store/eam-worker-configurate/actions.ts

```diff
@@ -17,13 +17,21 @@ const workerCreate = createAsyncThunk<
 >(
   ActionType.CREATE_WORKER,
   async (payload: EAMWorkerCreateRequestDto, { extra }) => {
-    const { workerApi, navigation, notification } = extra;
+    const { workerApi, navigation, notification, saveCsv } = extra;
+
+    const password = getRandomId();
 
     const worker = await workerApi.createWorker({
       ...payload,
-      password: getRandomId(),
+      password,
     });
 
+    const content = [
+      ['name:', payload.name],
+      ['password:', password],
+    ];
+    saveCsv.save(content, 'Credentials.csv');
```

Comment:
```suggestion
    saveCsv.save([
      ['name:', payload.name],
      ['password:', password],
    ], `${payload.name}-worker-credentials`);
```
also, add enums for the column names